lib/bark.ex

```diff
@@ -51,10 +51,10 @@ defmodule Bark do
   defp line_number(%{line: line}), do: line
   defp line_number(not_line), do: inspect(not_line)
 
-  # Quote string if it contains a space
+  # Quote string if it contains a space, and if there are quotes within in the string replace them with 🩹
   defp quote_if_spaces(value) when is_binary(value) do
     if String.contains?(value, " ") do
-      value_with_escaped_quotes = String.replace(value, "\"", "\\\"")
+      value_with_escaped_quotes = String.replace(value, "\"", "🩹")
```

Comment:
Can we just do an _ instead of a bandage emoji?
```suggestion
      value_with_escaped_quotes = String.replace(value, "\"", "_")
```

Reply:
Previous: (missing in result)
```
21:20:16.864 trace_id=1076151311172133351 span_id=1266269587818792854 [warn] module=Zipato.DeviceIntegration command=remove_access_code_from_device/2 error=error_removing_access_code reason=unknown slot_id=0 device_id=1345836 error_data="%{"command" => "SET", "status" => "NOTIFICATION_TIMEOUT", "success" => false, "transactionId" => "b39021bf-26b6-4a56-9854-3018d3f12dac", "type" => "USER_CODE_INPUT"}"
```
```json
{
  "timestamp": 1599600016864,
  "trace_id": 1076151311172133400,
  "span_id": 1266269587818793000,
  "module": "Zipato.DeviceIntegration",
  "command": "remove_access_code_from_device/2",
  "error": "error_removing_access_code",
  "reason": "unknown",
  "slot_id": 0,
  "device_id": 1345836
}
```

Stripped:
```
21:20:16.864 trace_id=1076151311172133351 span_id=1266269587818792854 [warn] module=Zipato.DeviceIntegration command=remove_access_code_from_device/2 error=error_removing_access_code reason=unknown slot_id=0 device_id=1345836 error_data="%{command => SET, status => NOTIFICATION_TIMEOUT, success => false, transactionId => b39021bf-26b6-4a56-9854-3018d3f12dac, type => USER_CODE_INPUT}"
```
```json
{
  "timestamp": 1599600016864,
  "trace_id": 1076151311172133400,
  "span_id": 1266269587818793000,
  "module": "Zipato.DeviceIntegration",
  "command": "remove_access_code_from_device/2",
  "error": "error_removing_access_code",
  "reason": "unknown",
  "slot_id": 0,
  "device_id": 1345836,
  "error_data": "%{command => SET, status => NOTIFICATION_TIMEOUT, success => false, transactionId => b39021bf-26b6-4a56-9854-3018d3f12dac, type => USER_CODE_INPUT}"
}
```


Replaced with _:
```
21:20:16.864 trace_id=1076151311172133351 span_id=1266269587818792854 [warn] module=Zipato.DeviceIntegration command=remove_access_code_from_device/2 error=error_removing_access_code reason=unknown slot_id=0 device_id=1345836 error_data="%{_command_ => _SET_, _status_ => _NOTIFICATION_TIMEOUT_, _success_ => false, _transactionId_ => _b39021bf-26b6-4a56-9854-3018d3f12dac_, _type_ => _USER_CODE_INPUT_}"
```
```json
{
  "timestamp": 1599600016864,
  "trace_id": 1076151311172133400,
  "span_id": 1266269587818793000,
  "module": "Zipato.DeviceIntegration",
  "command": "remove_access_code_from_device/2",
  "error": "error_removing_access_code",
  "reason": "unknown",
  "slot_id": 0,
  "device_id": 1345836,
  "error_data": "%{_command_ => _SET_, _status_ => _NOTIFICATION_TIMEOUT_, _success_ => false, _transactionId_ => _b39021bf-26b6-4a56-9854-3018d3f12dac_, _type_ => _USER_CODE_INPUT_}"
}
```


Replaced with junk:
```
21:20:16.864 trace_id=1076151311172133351 span_id=1266269587818792854 [warn] module=Zipato.DeviceIntegration command=remove_access_code_from_device/2 error=error_removing_access_code reason=unknown slot_id=0 device_id=1345836 error_data="%{🩹command🩹 => 🩹SET🩹, 🩹status🩹 => 🩹NOTIFICATION_TIMEOUT🩹, 🩹success🩹 => false, 🩹transactionId🩹 => 🩹b39021bf-26b6-4a56-9854-3018d3f12dac🩹, 🩹type🩹 => 🩹USER_CODE_INPUT🩹}"
```
```json
{
  "timestamp": 1599600016864,
  "trace_id": 1076151311172133400,
  "span_id": 1266269587818793000,
  "module": "Zipato.DeviceIntegration",
  "command": "remove_access_code_from_device/2",
  "error": "error_removing_access_code",
  "reason": "unknown",
  "slot_id": 0,
  "device_id": 1345836,
  "error_data": "%{🩹command🩹 => 🩹SET🩹, 🩹status🩹 => 🩹NOTIFICATION_TIMEOUT🩹, 🩹success🩹 => false, 🩹transactionId🩹 => 🩹b39021bf-26b6-4a56-9854-3018d3f12dac🩹, 🩹type🩹 => 🩹USER_CODE_INPUT🩹}"
}
```

Reply:
Replaced with ″:  (double prime)
```
21:20:16.864 trace_id=1076151311172133351 span_id=1266269587818792854 [warn] module=Zipato.DeviceIntegration command=remove_access_code_from_device/2 error=error_removing_access_code reason=unknown slot_id=0 device_id=1345836 error_data="%{″command″ => ″SET″, ″status″ => ″NOTIFICATION_TIMEOUT″, ″success″ => false, ″transactionId″ => ″b39021bf-26b6-4a56-9854-3018d3f12dac″, ″type″ => ″USER_CODE_INPUT″}"
```
```json
{
  "timestamp": 1599600016864,
  "trace_id": 1076151311172133400,
  "span_id": 1266269587818793000,
  "module": "Zipato.DeviceIntegration",
  "command": "remove_access_code_from_device/2",
  "error": "error_removing_access_code",
  "reason": "unknown",
  "slot_id": 0,
  "device_id": 1345836,
  "error_data": "%{″command″ => ″SET″, ″status″ => ″NOTIFICATION_TIMEOUT″, ″success″ => false, ″transactionId″ => ″b39021bf-26b6-4a56-9854-3018d3f12dac″, ″type″ => ″USER_CODE_INPUT″}"
}
```